### PR TITLE
DMP-3043: Transformed media search results

### PR DESCRIPTION
**Pull request by deanbullock-hmcts**
### Jira link (if applicable)

[DMP-3043](https://tools.hmcts.net/jira/browse/DMP-3043)

### CODE REVIEW

File: server/controllers/stub-data.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ const STUB_ALLOWED = config.get('allowStubData') === 'true';
 const STUB_RESPONSES: StubResponse[] = [
   // include stubs here like below
   // ...eventMappings,
+  // ...transformedMedia,
```
Comment (davet1985) [inline server/controllers/stub-data.ts:17]: Is this not stubbed?

**Review by davet1985 — COMMENTED** [review]
One question about the stubbing.

I think the courthouse search field should be using the `app-courthouse-field` component, but I'll check that.

File: server/controllers/stub-data.ts (hunk 1 above)
Comment (deanbullock-hmcts) [reply server/controllers/stub-data.ts:17]: Addressed in https://github.com/hmcts/darts-portal/pull/1118

**Review by DavidTMann — COMMENTED** [review]
Just a couple discrepancies in table column headers from Figma
- Media iD, Case ID, Date requested

**Review by DavidTMann — APPROVED** [review]

**Pull request merged**